crates/cli/src/main.rs

```diff
@@ -281,7 +296,10 @@ async fn main() -> Result<(), AnvilZksyncError> {
     // during replay. Otherwise, replay would send commands and hang.
     tokio::spawn(async move {
         if let Err(err) = node_executor.run().await {
-            sh_err!("node executor ended with error: {:?}", err);
+            let error = err.context("Node executor ended with error");
+            sh_err!("{:?}", error);
+
+            let _ = telemetry.track_error(Box::new(error.as_ref())).await;
```

Comment:
In a subsequent PR, can we track HALT, and REVERT errors from places like here: https://github.com/matter-labs/anvil-zksync/blob/055cd432d07202edfc5550edf86841fe165bdab7/crates/core/src/node/inner/in_memory_inner.rs#L929

Reply:
We can, but not sure if we want to, in most cases it should be just normal behaviour.

Reply:
Wouldnt this be great insight for us? What errors users are hitting most during development?

Reply:
Not sure tbh, could be, if filter out 99% of spam, we can definitely try and see how it goes.